### PR TITLE
feat(gatsby-plugin-gatsby-cloud): Remove preview UI

### PR DESCRIPTION
**Pull request by leithonenglish**
## Description

Removal of `Preview UI` code from `gatsby-plugin-gatsby-cloud` since `Preview UI` will be hosted within its own package here: https://www.npmjs.com/package/@gatsby-cloud-pkg/gatsby-plugin-preview.

Types of files removed:

- Components
- Tests for components
- Scripts building components
- Constants that affect the UI
- `gatsby-browser.js` in `src` and in `__tests__`
- Any packages relating to tests or that were used in components
- Minor updates to `babelrc` and the `gitignore` files

### CODE REVIEW

**Review by TylerBarnes — COMMENTED** [review]
Looks good to me skimming through the deleted files. I think lets just make sure we have the Cloud side figured out and merged before we merge this 👍

File: packages/gatsby-plugin-gatsby-cloud/.babelrc
Comment (wardpeet) [inline packages/gatsby-plugin-gatsby-cloud/.babelrc:8]: Remove the full overrides

**Review by DanielSLew — APPROVED** [review]

**Comment by wardpeet** [comment]
Can we get a canary out and make sure it doesn't break anything?

**Review by TylerBarnes — APPROVED** [review]
Failing tests look unrelated

**Comment by TylerBarnes** [comment]
Can you try updating `gatsby` to the latest version? There was a fix that went out that should resolve it

**Comment by apotheosistech** [comment]
Just another set of eyes here, but I too am on workspaces. I did update `gatsby-plugin-gatsby-cloud` but I noticed my .lock file is still showing 4.15.0.

**Comment by TylerBarnes** [comment]
A fix for this issue with workspaces and the preview plugin was released in the last hour. @apotheosistech @laurenskling can you confirm your builds are succeeding again?

**Comment by apotheosistech** [comment]
My builds are succeeding however previews do not work. I receive "404 openresty" errors. I'm not sure if this is related to the same issue though?

**Comment by TylerBarnes** [comment]
@apotheosistech can you open a Gatsby Cloud support ticket and add a link in the ticket to your comments here? I believe it's unrelated to this issue, though I could be wrong. Either way a support ticket will allow us to more easily debug the issue and look at your site.

**Comment by laurenskling** [comment]
Thanks for the assistance @TylerBarnes , it has started working again.

Sorry @apotheosistech , my previews are working, so this indeed seems another issue.

**Pull request merged**
